pages/homePage/homePage.css

```diff
@@ -0,0 +1,204 @@
+@import "/styles.css";
+
+.page-wrap {
+
+    grid-template-areas:
+        'p_nav p_nav p_nav p_nav p_nav p_nav'
+        'p_content p_content p_content p_content p_content p_content'
+        'p_content p_content p_content p_content p_content p_content'
+    ;
```

Comment:
No need for repetition. You can do:

"p_nav"
"p_content"

Reply:
Above, suggested format will work.

For now, I am keeping 6 columns for consistency with other pages and to tinker around, same tinkering is the reason for two p_content rows.